src/lang/lang_builtins.ml

```diff
@@ -141,8 +141,8 @@ let () =
   let proto =
     [
       ( "id",
-        Lang.string_t,
-        Some (Lang.string ""),
+        Lang.nullable_t Lang.string_t,
```

Comment:
I was also tempted by this change, but didn't dare ;)

Reply:
Well I was on a wild run :-)